fails:Kernel#raise inside a rescue block ignores inner exceptions
